packages/inspector/client/components.d.ts

```diff
@@ -1,6 +1,6 @@
 // generated by unplugin-vue-components
 // We suggest you to commit this file into source control
-// Read more: https://github.com/vuejs/core/pull/3399
+// Read more: https://github.com/vuejs/vue-next/pull/3399
```

Comment:
You are using an old local dep

Reply:
This file was auto generated, i don't know the reason about this change. I'm changed it manually now. I will check upstream code about `unplugin-vue-components`.